### PR TITLE
Use rust 1.37.0

### PR DESCRIPTION
**Pull request by sgkim126**

### CODE REVIEW

File: 3 [hunk 1]
```diff
@@ -0,0 +1,553 @@
+// Copyright 2019 Kodebox, Inc.
```
Comment (majecty) [inline 3]: Why is this file added?

Comment (sgkim126) [reply 3]: Mistake. I updated the PR.

**Comment by sgkim126** [comment]
@kseo Please review it.
